### DISCUSSION TITLE
Can't update bios

### DISCUSSION THREAD

**Discussion by Tresillo2017** [terminatorul/NvStrapsReBar#94] [General]
So while viewing this guide and ReBarUEFI I accidentaly patched the wrong file onto my BIOS `ReBarDxe.ffs` instead of `NvStrapsReBar.ffs`. Now when I try to patch again the bios with the other file I got an error saying that the bios os not correct. Even thought it worked the first time. I tried with the official one unpached and I get the same error.

Motherboard: MSI B450 Gaming Plus Max
GPU: Gigabyte RTX 2060 6GB
CPU: AMD Ryzen 5 3600x

![20240626_144900](https://github.com/terminatorul/NvStrapsReBar/assets/56476330/518efc8b-eca8-418c-94f4-9c084cdaa685)

**Comment by D337z** [comment]
First off, make 100% sure that you have the absolute correct UEFI/BIOS file (stock) and then try to use AFUWin64, if you can get it, from inside Windows.  If bad comes to worse, you may need to go the Raspberry Pi route and directly flash the chip.

**Reply by Tresillo2017** [reply]
I was able to patch it by using a newer bios version, which came a week ago
